docs/marionette.region.md

```diff
@@ -314,6 +316,60 @@ mainRegion.hasView() // true
 If you show a view in a region with an existing view, Marionette will
 [remove the existing View](#emptying-a-region) before showing the new one.
 
+### Non-Marionette Views
+
+Marionette Regions aren't just for showing Marionette Views - they can also
+display instances of regular [`Backbone.View`](http://backbonejs.org/#View).
+To do this, ensure your view defines a `render()` method and just treat it like
+a regular Marionette View:
+
+```javascript
+var Bb = require('backbone');
+var Mn = require('backbone.marionette');
+var _ = require('underscore');
+
+var MyChildView = Bb.View.extend({
+  render: function() {
+    this.$el.append('<p>Some text</p>');
+  }
+});
+
+var MyParentView = Mn.View.extend({
+  regions: {
+    child: '.child-view'
+  },
+
+  template: _.template('<div class="child-view"></div>'),
+
+  onRender: function() {
+    this.showChildView('child', new MyChildView());
+  }
+});
+```
+
+#### Showing Backgrid Views
```

Comment:
That makes sense, though I think it's important to reference specific libraries that do this here. My aim is to ensure users of libraries like Backgrid can just see straight away what they need to do without digging into the internals of how both Backgrid and Marionette work.

I'll come up with a better way of wording it for the docs.